android/BOINC/app/src/main/res/layout/projects_layout_listitem.xml

```diff
@@ -110,6 +110,31 @@
                     android:text="@string/projects_credits_host_and_user"
                     android:textAppearance="?android:attr/textAppearanceSmall"
                     android:textColor="?android:attr/textColorPrimary"/>
+            </LinearLayout>
```

Comment:
Please fix indents

Reply:
Please look at whole file. This is the correct indent